Inbox docs update (#692)
novuhq/docs: inbox/react/components/inbox.mdx

---
title: '<Inbox />'
---

> It renders an opinionated `<Inbox />` with a bell button, that triggers a popover on lock. The popover contains the notifications list and the user preferences.

Reviewer: ```suggestion
Renders `<Inbox />` with a bell button, that opens a popover on click. The popover contains the notifications list and the user preferences.
```
